src/Compiler.php

```diff
@@ -271,7 +271,7 @@ protected function compileServers($value)
     {
         $pattern = $this->createMatcher('servers');
 
-        return preg_replace($pattern, '$1<?php $__container->servers$2; ?>', $value);
+        return preg_replace($pattern.'s', '$1<?php $__container->servers$2; ?>', $value);
```

Comment:
The `s` flag causes the `.` in the pattern to match all characters, instead of all characters except line breaks.